tests/soc_descs/blackhole_140_arch.yaml

```diff
@@ -0,0 +1,72 @@
+# Note taken from software repo - may need updates.
+grid:
+  x_size: 17
+  y_size: 12
+
+arc:
+  [ 8-0 ]
+
+pcie:
+  [ 11-0 ]
+
+dram:
+  [
+      [0-0, 0-1, 0-11],
+      [0-2, 0-10, 0-3],
+      [0-9, 0-4, 0-8],
+      [0-5, 0-7, 0-6],
+      [9-0, 9-1, 9-11],
+      [9-2, 9-10, 9-3],
+      [9-9, 9-4, 9-8],
+      [9-5, 9-7, 9-6],
+  ]
+
+eth:
+  [
+   1-1, 2-1, 3-1, 4-1, 5-1, 6-1, 7-1, 10-1, 11-1, 12-1, 13-1, 14-1, 15-1, 16-1,
+  ]
+
+functional_workers:
+  [
+   1-2,   2-2,   3-2,   4-2,   5-2,   6-2,   7-2,   10-2,   11-2,   12-2,   13-2,   14-2,   15-2,   16-2,
+   1-3,   2-3,   3-3,   4-3,   5-3,   6-3,   7-3,   10-3,   11-3,   12-3,   13-3,   14-3,   15-3,   16-3,
+   1-4,   2-4,   3-4,   4-4,   5-4,   6-4,   7-4,   10-4,   11-4,   12-4,   13-4,   14-4,   15-4,   16-4,
+   1-5,   2-5,   3-5,   4-5,   5-5,   6-5,   7-5,   10-5,   11-5,   12-5,   13-5,   14-5,   15-5,   16-5,
+   1-6,   2-6,   3-6,   4-6,   5-6,   6-6,   7-6,   10-6,   11-6,   12-6,   13-6,   14-6,   15-6,   16-6,
+   1-7,   2-7,   3-7,   4-7,   5-7,   6-7,   7-7,   10-7,   11-7,   12-7,   13-7,   14-7,   15-7,   16-7,
+   1-8,   2-8,   3-8,   4-8,   5-8,   6-8,   7-8,   10-8,   11-8,   12-8,   13-8,   14-8,   15-8,   16-8,
+   1-9,   2-9,   3-9,   4-9,   5-9,   6-9,   7-9,   10-9,   11-9,   12-9,   13-9,   14-9,   15-9,   16-9,
+   1-10,  2-10,  3-10,  4-10,  5-10,  6-10,  7-10,  10-10,  11-10,  12-10,  13-10,  14-10,  15-10,  16-10,
+   1-11,  2-11,  3-11,  4-11,  5-11,  6-11,  7-11,  10-11,  11-11,  12-11,  13-11,  14-11,  15-11,  16-11,
+ ]
+  
+harvested_workers:
+  []
+
+router_only:
+  [
+   1-0, 2-0, 3-0, 4-0, 5-0, 6-0, 7-0, 10-0, 12-0, 13-0, 14-0, 15-0, 16-0,
+   8-1, 8-2, 8-3, 8-4, 8-5, 8-6, 8-7, 8-8, 8-9, 8-10, 8-11
+  ]
+
+worker_l1_size:
+  1499136
+
+dram_bank_size:
+  2147483648
```

Comment:
should this be 4GB?

Reply:
Should be, thanks for catching. 
I've fixed it.